elements/designer-stage/designer-stage.html

```diff
@@ -425,10 +429,19 @@
             return;
         }
 
-        this.frame.client.moveCursor(move)
-          .then(function(response) {
-            this._updateCaret(response.rect);
-          }.bind(this));
+        if (move != 'backspace') {
```

Comment:
let's swap these cases so the default is a move: `if (move === 'backspace') {...`